wasm/smolmix/README.md

```
@@ -0,0 +1,144 @@
+## Public API
+
+Two WASM exports that mirror the browser's native networking surface:
+
+| Browser API | smolmix export | Description |
+|-------------|---------------|-------------|
+| `fetch()` | `mixFetch(url, init)` | HTTP/HTTPS request-response |
+| `new WebSocket()` | `mixSocket(url, protocols, onEvent)` | WebSocket (WS/WSS) |
+
+## Arch
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)